src/screens/DiscoveryScreenV2/index.js

```diff
@@ -311,13 +311,13 @@ const DiscoveryScreenV2 = ({route}) => {
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: COLORS.white,
+    backgroundColor: '#fff',
     flex: 1,
     paddingTop: 60
   },
   fragmentContainer: {
     height: '100%',
-    backgroundColor: COLORS.white
+    backgroundColor: colors.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value for `backgroundColor` in the `container` and `fragmentContainer` styles has been changed from `COLORS.white` to `colors.white`. Ensure that the new `colors.white` value is equivalent to the old `COLORS.white` value, otherwise it may cause visual inconsistencies in your application.
```diff
-    backgroundColor: COLORS.white,
+    backgroundColor: colors.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->